resources/lang/es.json

```diff
@@ -271,29 +271,29 @@
 	"Number": "Número",
 	"O'clock": "En punto",
 	"Once a invoice has been send, no new invoice lines can be added": "Una vez que se ha enviado una factura, no se pueden agregar nuevas líneas",
-	"Online Users": "Usuarios en línea",
+	"Online Users": "Usuarios en Línea",
 	"Only Allowed for admins": "Solo permitido para administradores",
 	"Only Allowed for the user who registered the CRM": "Solo permitido para el usuario que registró el CRM.",

```

Comment:
"Solo" means Alones, Sólo (Solamente) is the right one for only.

Reply:
https://www.rae.es/consultas/el-adverbio-solo-y-los-pronombres-demostrativos-sin-tilde#:~:text=La%20palabra%20solo%2C%20tanto%20cuando,Quiero%20aquella)%20o%20como%20determinantes%20(

Solo is the only correct word right now, you shouldn't use Sólo no more (deprecated xd)

Reply:
Woah, is that new? 
https://www.rae.es/dpd/tilde#323
3.2.3. sólo/solo. La palabra solo puede ser un adjetivo: No me gusta el café solo; Vive él solo en esa gran mansión; o un adverbio: Solo nos llovió dos días; Contesta solo sí o no. Se trata de una palabra llana terminada en vocal, por lo que, según las reglas generales de acentuación (→ 1.1.2), no debe llevar tilde. Ahora bien, cuando esta palabra pueda interpretarse en un mismo enunciado como adverbio o como adjetivo, se utilizará obligatoriamente la tilde en el uso adverbial para evitar ambigüedades: Estaré solo un mes (al no llevar tilde, solo se interpreta como adjetivo: ‘en soledad, sin compañía’); **Estaré sólo un mes (al llevar tilde, sólo se interpreta como adverbio: ‘solamente, únicamente’); también puede deshacerse la ambigüedad sustituyendo el adverbio solo por los sinónimos solamente o únicamente.**

Is DRAE contradicting itself?

Reply:
But well if it's a new rule, let's keep it easier.

Reply:
It was a very controversial rule for sure! Will correct the spaces in a bit 😘